pyenvisalink/alarm_panel.py

```diff
@@ -356,44 +365,44 @@ async def discover_device_details(self) -> bool:
 
         try:
             async with aiohttp.ClientSession(auth=aiohttp.BasicAuth(self._username, self._password)) as client:
-                url = f'http://{self._host}:{self._httpPort}/2'
+                url = f"http://{self._host}:{self._httpPort}/2"
                 resp = await client.get(url)
                 if resp.status != 200:
                     _LOGGER.warn("Unable to discover Envisalink version and patel type: '%s'", resp.status)
                     return False
 
                 # Try and scrape the HTML for the EVL version and panel type
                 html = await resp.text()
-                version_regex = '<TITLE>Envisalink ([0-9])<\/TITLE>'
+                version_regex = r"<TITLE>Envisalink ([0-9])<\/TITLE>"
 
                 m = re.search(version_regex, html)
                 if m is None or m.lastindex != 1:
                     _LOGGER.warn("Unable to determine version: raw HTML: %s", html)
                 else:
                     self._evlVersion = int(m.group(1))
 
-                panel_regex = '>Security Subsystem - ([^<]*)<'
+                panel_regex = ">Security Subsystem - ([^<]*)<"
                 m = re.search(panel_regex, html)
                 if m is None or m.lastindex != 1:
                     _LOGGER.warn("Unable to determine panel type: raw HTML: %s", html)
                 else:
                     self._panelType = m.group(1).upper()
-                    if self._panelType not in [ PANEL_TYPE_DSC, PANEL_TYPE_HONEYWELL ]:
+                    if self._panelType not in [PANEL_TYPE_DSC, PANEL_TYPE_HONEYWELL]:
                         _LOGGER.warn("Unrecognized panel type: %s", self._panelType)
         except Exception as ex:
             _LOGGER.error("Unable to fetch panel information: %s", ex)
             return self.ConnectionResult.CONNECTION_FAILED
 
-        _LOGGER.info(f"Discovered Envisalink %s: %s", self._evlVersion, self._panelType)
+        _LOGGER.info(f"Discovered Envisalink {self._evlVersion}: {self._panelType}")
```

Comment:
I notice some places you've updated logging to use f-strings but not in others.  Is there a criteria you are using?

Home Assistant has guidance not to use f-strings for logging to avoid formatting strings when logging is suppressed - which I think is generally a good thing so had planned to switch all the logging to that and away from the str.format() stuff that's mostly in this code.

Reply:
Fixed all these.